static_src/actions/app_actions.js

```diff
@@ -54,27 +74,35 @@ export default {
       appGuid,
       app
     });
+
+    return Promise.resolve(app);
```

Comment:
There's no hard rule. For fetch actions, it should be the entity that is being fetched. For receive actions, it should be the input so that it is chainable. The exception is error actions, I think it's best to return void here although I haven't made a final opinion on it.